src/dashboard/src/main/management/commands/update_elasticsearch_mappings.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""Update Elasticsearch mappings via Put Mapping.
+
+Updates the Elasticsearch mappings for the aips and aipfiles indices for
+Archivematica 1.12 to enable sorting on all fields displayed in the new
+Archival Storage DataTable.
+
+Execution example:
+
+./manage.py update_elasticsearch_mappings
+"""
+from __future__ import absolute_import, print_function
+
+import sys
+
+from django.conf import settings
+from elasticsearch import ElasticsearchException
+
+from main.management.commands import DashboardCommand
+import elasticSearchFunctions as es
+
+
+class Command(DashboardCommand):
+
+    help = __doc__
+
+    def handle(self, *args, **options):
+        # Check that AIPs index is enabled
+        if es.AIPS_INDEX not in settings.SEARCH_ENABLED:
+            self.error(
+                "The AIPs indexes are not enabled. Please, make sure to "
+                "set the *_SEARCH_ENABLED environment variables to `true` "
+                "to enable the AIPs and Transfers indexes, or to `aips` "
+                "to only enable the AIPs indexes."
+            )
+            sys.exit(1)
+
+        es.setup_reading_from_conf(settings)
+        es_client = es.get_client()
+
+        try:
+            es_client.info()
+        except ElasticsearchException:
+            print("Error: Elasticsearch may not be running.")
+            sys.exit(1)
+
+        # Update AIPs index mappings
+        es_client.indices.put_mapping(
+            index=es.AIPS_INDEX,
+            doc_type=es.DOC_TYPE,
+            body={
+                "properties": {
+                    "accessionids": {"type": "keyword"},
+                    "status": {"type": "keyword"},
+                    "file_count": {"type": "integer"},
```

Comment:
We really need to normalize field names in our indexes, but it may be to late to it in here. I added a comment about this in ... https://github.com/archivematica/Issues/issues/404.

Reply:
Yes, agreed @jraddaoui! I tried to balance consistency in the indices with keeping a reasonable scope for the project (especially given that the scope of work involved for the Archival Storage tab was larger than estimated) and not requiring too many changes to existing installations, but in retrospect I would have focused more on normalizing field names. And now that I'm more familiar with Elasticsearch, I see that `accessionids` for example could have just been `accessionid`, since ES will natively handle turning a field into an array if there are multiple values of the same type (and of course, ideally would be `accession_id` across the different indices).

Thanks for adding the comment to https://github.com/archivematica/Issues/issues/404